pulsar-io/redis/src/main/java/org/apache/pulsar/io/redis/sink/RedisSink.java

```diff
@@ -119,7 +119,7 @@ private void flush() {
         }
 
         if (CollectionUtils.isNotEmpty(recordsToFlush)) {
-            for (Record<T> record: recordsToFlush) {
+            for (Record<byte[]> record: recordsToFlush) {
                 try {
                     // records with null keys or values will be ignored
                     byte[] key = toBytes("key", record.getKey().orElse(null));
```

Comment:
I think we can get rid of `toBytes` here, no?

Reply:
yeah, it's redundant in this case. Changed.